README.md

```diff
@@ -1,54 +1,20 @@
-I would like to see a list of dishes with prices
+Adding a dish to the basket. order.add("pizza", 3)
+If the dish is not in the menu it will show a message.
+If you add two times the the same dish, it will only add to the basket the quantity you specified the second time.
 
-As a customer
-So that I can order the meal I want
-I would like to be able to select some number of several available dishes
+Check your basket: order.basket
 
-As a customer
-So that I can verify that my order is correct
-I would like to check that the total I have been given matches the sum of the various dishes in my order
+Checking the total amount of your basket: order.total
 
-As a customer
-So that I am reassured that my order will be delivered on time
-I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
-```
+Checkout: order.checkout(amount)
+If the total amount is different from the amount given on checkout, it will show an error message.
 
-* Hints on functionality to implement:
-  * ensure you have a list of dishes with prices
-  * place the order by giving the list of dishes, their quantities and a number that should be the exact total. If the sum is not correct the method should raise an error, otherwise the customer is sent a text saying that the order was placed successfully and that it will be delivered 1 hour from now, e.g. "Thank you! Your order was placed and will be delivered before 18:52".
-  * The text sending functionality should be implemented using Twilio API. You'll need to register for it. It’s free.
-  * Use the twilio-ruby gem to access the API
-  * Use a Gemfile to manage your gems
-  * Make sure that your Takeaway is thoroughly tested and that you use mocks and/or stubs, as necessary to not to send texts when your tests are run
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
-
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
-* Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
-
-
-**Note: We are looking for good OO design and programming! Remember the Single Responsibility and Dependency Injection/Inversion principles!**
-
-[![Build Status](https://travis-ci.org/makersacademy/takeaway-challenge.svg?branch=master)](https://travis-ci.org/makersacademy/takeaway-challenge)
-[![Coverage Status](https://coveralls.io/repos/makersacademy/takeaway-challenge/badge.png)](https://coveralls.io/r/makersacademy/takeaway-challenge)
+I didn't have time to implement the text sending functionality with twilio.
```

Comment:
In the readme, I would like to see:

- The brief
- The user stories
- The learning objectives
- How much coding experience you have at the time
- How long it took you to build i.e. a weekend
- How to install (in this case: clone from github, run bundle)
- How to start the server ( not relevant to this challenge, but will be relevant from now onwards)
- How to run tests (in this case run RSpec)

Keep it concise and informative :)